build/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.10
+
+# Some ENV variables
+ENV PATH="/mattermost/bin:${PATH}"
+ARG PUID=2000
+ARG PGID=2000
+ARG MM_PACKAGE=
```

Comment:
If we have this in the main repo are people going to expect a docker build and run to just work? It won't because of this.

Reply:
fixed